nix/README.md

```diff
@@ -1,29 +1,35 @@
 ### Nix support
 
+### Quickstart
```

Comment:
What is Nix and why are we using it? Is it going to make my development workflow easier, and if yes, how? Does it replace Docker? (sort of?) Does it replace cmake? (no?)